collections/_patterns/payment.html

```diff
@@ -75,4 +75,14 @@ <h2 class="title fw-bold" id="guidelines">Guidelines</h2>
         </p>
     </section>
 
-</div>
\ No newline at end of file
+</div>
```

Comment:
lets say a user type 12/30 and decided to press backspace.. 

able to detect backspace? and remove "/" when "30" is deleted?